### PR TITLE
* #25 - USER_PERMISSION_DENIED. The loginCustomerId header is not added

### PR DESCRIPTION
**Pull request by imerenanu**
- handles login-customer-id in GoogleAdsGapicClientTrait, passes it further to FixedHeaderMiddleware;

### CODE REVIEW

File: src/Google/Ads/GoogleAds/Lib/GoogleAdsGapicClientTrait.php [hunk 1]
```diff
@@ -46,9 +51,12 @@ protected function modifyClientOptions(array &$options)
     protected function modifyUnaryCallable(callable &$callable)
     {
         if (!is_null($this->developerToken)) {
-            $callable = new FixedHeaderMiddleware($callable, [
-                self::$DEVELOPER_TOKEN_KEY => [$this->developerToken]
-            ]);
+            $headers = [self::$DEVELOPER_TOKEN_KEY => [$this->developerToken]];
+
+            if (!is_null($this->loginCustomerId))
```
Comment (fiboknacky) [inline src/Google/Ads/GoogleAds/Lib/GoogleAdsGapicClientTrait.php]: Could you add a brace around this if clause, per PSR-2 standard?

**Comment by imerenanu** [comment]
Hi @fiboknacky,

PSR-2 respected.

**Comment by fiboknacky** [comment]
> Hi @fiboknacky,
> 
> PSR-2 respected.

Thank you. Could you take care of CLA check too?

**Comment by imerenanu** [comment]
Hi @fiboknacky ,

I have submitted CLA agreement.
I did it behalf of Employer, don't know when it going to be active.

**Comment by imerenanu** [comment]
I signed it!

**Comment by fiboknacky** [comment]
> Hi @fiboknacky ,
> 
> I have submitted CLA agreement.
> I did it behalf of Employer, don't know when it going to be active.

Could you reply to GoogleBot as "I signed it!"?

**Comment by imerenanu** [comment]
I signed it!

**Comment by imerenanu** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).

**Comment by imerenanu** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).

I signed it!

**Comment by fiboknacky** [comment]
Hi @imerenanu 

Is it intentional to close this PR? 
I see that you've signed CLA already, so it's ready to be merged.

Knack

**Comment by imerenanu** [comment]
Hi @fiboknacky ,

No, I did it by mistake. Don't know how it happened :)

**Comment by fiboknacky** [comment]
Thanks for confirming.

**Pull request merged**